.env

```diff
@@ -0,0 +1,9 @@
+DEBUG=1
+SECRET_KEY='7*fsmud8l*&)o^m2jp4$fn%nk$g)xh9z$y+q96_uj_+vmvirdj'
```

Comment:
lol is this a real secret key. i assume all of these things are for local .env

Reply:
Yep, just for local dev